readme.md

```diff
@@ -31,7 +31,7 @@ false
 
 ## API
 
-### firstRun([options])
+### firstRun({options})
```

Comment:
`[]` doesn't mean array. It's the convention for signifying an optional argument.